subprojects/docs/src/samples/testKit/gradleRunner/automaticClasspathInjectionCustomTestSourceSet/groovy/build.gradle

```diff
@@ -10,7 +12,7 @@ sourceSets {
         resources {
             srcDir file('src/functionalTest/resources')
         }
-        compileClasspath += sourceSets.main.output + configurations.testRuntime
+        compileClasspath += sourceSets.main.output + configurations.testRuntimeClasspath
```

Comment:
Note to reviewer: I changed this in order to avoid using the deprecated testRuntime configuration, but I'm not sure that it is correct. If not, please tell.
Note to myself: if incorrect, the kotlin build script should be modified, too.